test/DartTest.ts

```diff
@@ -201,8 +201,11 @@ describe('transpile to dart', () => {
     it('special cases @CONST', () => {
       expectTranslate('@CONST class X {}').to.equal(' const class X { }');
       expectTranslate('@CONST() class X {}').to.equal(' const class X { }');
-      expectTranslate('class X { @CONST constructor() { super(3); this.x = 1; this.y = 2; } }')
-          .to.equal(' class X { const X ( ) : x = 1 , y = 2 , super ( 3 ) ; }');
+      expectTranslate('class X { ' +
+                      '  x: number; ' +
+                      '  @CONST constructor() { super(3); this.x = 1; this.y = 2; } ' +
+                      '}')
```

Comment:
nit: this would be better as a separate assertion with just the member and no statements in the ctor body